hq/conf/application.conf

```diff
@@ -58,6 +58,10 @@ anghammaradSnsArn = ${ANGHAMMARAD_SNS_TOPIC_ARN}
 
 iamDynamoTableName = ${IAM_DYNAMO_TABLE_NAME}
 
+iamUnrecognisedUserS3Bucket = ${IAM_UNRECOGNISED_USER_S3_BUCKET}
```

Comment:
Now there are a few of these, is it worth grouping them for clarity? There's no practical difference in having them be grouped so no need unless you it makes this file easier to read and understand.